entities/entities/spawned_money/cl_init.lua

```diff
@@ -3,27 +3,25 @@ include("shared.lua")
 function ENT:Initialize()
 end
 
+local vector_up = Vector(0,0,1)
```

Comment:
That is just to isolate the "z" value I get from the objects bounding box. (Which is done a few lines down)